client/src/components/landing-components/Hero.jsx

```diff
@@ -1,95 +1,62 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import Paper from '@mui/material/Paper';
 import Typography from '@mui/material/Typography';
 import Grid from '@mui/material/Grid';
-import Link from '@mui/material/Link';
 import Box from '@mui/material/Box';
-import { Button, Shadows, alpha } from '@mui/material';
-import heroImg from "../../assets/img/heroImage.jpg";
+import { Button } from '@mui/material';
 import { tokens } from "../../theme";
 import { useTheme } from "@emotion/react";
 import useColorTheme from "../../hooks/FormStyles";
 import { useNavigate } from 'react-router-dom';
+import HeroStyles from '../../hooks/HeroStyles';
 
 function Hero(props) {
   const theme = useTheme();
   const colors = tokens(theme.palette.mode);
   const colorTheme = useColorTheme();
   const { post } = props;
   const navigate = useNavigate();
+  const heroStyle = HeroStyles();
```

Comment:
This should probably match the s on the end for consistency.

Reply:
Okay